i18n/en.json

```diff
@@ -4142,6 +4142,29 @@
   "postlist.toast.searchHint": "Tip: Try {searchShortcut} to search this channel",
   "posts_view.loadMore": "Load More messages",
   "posts_view.newMsg": "New Messages",
+  "pricing_modal.briefing.enterprise": "All the features of Professional, plus",
+  "pricing_modal.briefing.enterprise.advancedCompliance": "Advanced compliance",
+  "pricing_modal.briefing.professional": "All the features of Starter, plus",
+  "pricing_modal.briefing.professional.advancedRolesAndPermissions": "Advanced roles and permissions",
+  "pricing_modal.briefing.professional.highAvailability": "High Availability",
+  "pricing_modal.briefing.professional.readOnlyAnnoucementChannels": "Read-only announcement channels",
+  "pricing_modal.briefing.starter": "Starter comes with...",
+  "pricing_modal.briefing.starter.boardCards": "{boards} Boards cards per server",
+  "pricing_modal.briefing.starter.messageHistory": "Limited message history",
+  "pricing_modal.briefing.starter.teamCount": "Limited to one team",
+  "pricing_modal.briefing.storage": "{storage}GB file storage",
+  "pricing_modal.btn.contactSales": "Contact Sales",
+  "pricing_modal.btn.tryDays": "Try free for {days} days",
+  "pricing_modal.btn.upgrade": "Upgrade",
+  "pricing_modal.lookingToSelfHost": "Looking to self-host?",
+  "pricing_modal.planDisclaimer.starter": "This plan has data restrictions.",
+  "pricing_modal.planLabel.currentPlan": "CURRENT PLAN",
+  "pricing_modal.planLabel.mostPopular": "MOST POPULAR",
+  "pricing_modal.price.contactSales": "Contact sales",
+  "pricing_modal.price.free": "Free",
+  "pricing_modal.rate.userPerMonth": "/user/month",
+  "pricing_modal.reviewDeploymentOptions": "Review deployment options",
+  "pricing_modal.title": "Select a plan",
   "promote_to_user_modal.desc": "This action promotes the guest {username} to a member. It will allow the user to join public channels and interact with users outside of the channels they are currently members of. Are you sure you want to promote guest {username} to member?",
```

Comment:
```suggestion
  "promote_to_user_modal.desc": "This action promotes the guest {username} to a member. They'll be able to join public channels and interact with other users outside of the channels they're currently members of. Are you sure you want to promote guest {username} to member?",
```

Reply:
I've edited this pending confirmation of my assumption of the behavior: 

When a user is promoted to guest, they can join public channels. Are they able to interact with other, non-guest, users outside of the channels that the guest user is (as in via DM or GM)? Basically they're limited to certain channels but not limited to who they can engage with directly?

Reply:
@justinegeffen maybe we can fix this in another issue.

Reply:
+1 on limiting this PR to pricing modal changes - We are not sure either about the expected behavior for guests

Reply:
Thanks, @AGMETEOR and @marianunez, I'll create a separate PR for this. :)